pywbem/cim_operations.py

```diff
@@ -861,6 +890,100 @@ def __init__(self, url, creds=None, default_namespace=DEFAULT_NAMESPACE,
         self._last_server_response_time = None
 
     @property
+    def operation_recorder(self):
+        """
+        This is a writeable property; setting this property will cause the
+        defined recorder to be activated.
+
+        Effective pywbem 0.10.0 operation_recorder became a property rather
+        than an attribute. The getter operation returns TODO
+        The setting operation executes the add_operation_recorder with the
```

Comment:
If we mention the method, do it with a link.

Reply:
Done.